src/IssueViz.Security.UnitTests/HotspotsList/HotspotTableEntryWpfElementFactoryTests.cs

```diff
@@ -27,17 +26,55 @@
 using FluentAssertions;
 using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.Text;
 using Moq;
+using SonarLint.VisualStudio.IssueVisualization.Models;
 using SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource;
 using DataTrigger = System.Windows.DataTrigger;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.HotspotsList
 {
     [TestClass]
-    public class NonNavigableFrameworkElementFactoryTests
+    public class HotspotTableEntryWpfElementFactoryTests
     {
         [TestMethod]
-        public void Create_CreatesStyledElement()
+        public void Create_CreatesElementWithContextMenu()
```

Comment:
Strictly speaking this only tests that navigable elements have context menus. It wouldn't hurt to make it data driven so it checks both states.